src/ontology/mondo-edit.obo

```diff
@@ -393520,10 +393520,11 @@ synonym: "GEFS+" EXACT [DOID:0060170, Orphanet:36387]
 synonym: "generalised epilepsy with febrile seizures-plus" RELATED OMO:0003005 []
 synonym: "generalized epilepsy with febrile seizures plus" EXACT CLINGEN_LABEL []
 synonym: "generalized epilepsy with febrile seizures-plus" RELATED [Orphanet:36387]
-synonym: "genetic epilepsy with febrile seizures plus" EXACT CLINGEN_LABEL [https://www.clinicalgenome.org/affiliation/40005/, Orphanet:36387]
+synonym: "genetic epilepsy with febrile seizures plus" EXACT CLINGEN_LABEL [https://www.clinicalgenome.org/affiliation/40005/, https://www.epilepsydiagnosis.org/syndrome/fbp-overview.html, Orphanet:36387]
```

Comment:
@matentzn @twhetzel - the qc is failing because of this xref to this URL: https://www.epilepsydiagnosis.org/syndrome/fbp-overview.html

are we not allowed to xref a URL on a synonym?